Cargo.toml

```diff
@@ -20,19 +20,31 @@ include = [
   "data/",
   "csrc/",
   "src/",
-  "submodules/downloads",
+  "submodules/downloads/ruy",
```

Comment:
If there are things in downloads that we don't need, we should drop them in the script, and keep this simple.